pallets/registrar/src/lib.rs

```diff
@@ -441,6 +442,37 @@ pub mod pallet {
 
             Ok(())
         }
+
+        /// Pause container-chain from collating without removing its boot nodes nor its genesis config
+        #[pallet::call_index(4)]
+        #[pallet::weight(T::WeightInfo::pause_container_chain(T::MaxLengthParaIds::get()))]
+        pub fn pause_container_chain(origin: OriginFor<T>, para_id: ParaId) -> DispatchResult {
+            T::RegistrarOrigin::ensure_origin(origin)?;
+
+            let mut pending_verification = PendingVerification::<T>::get();
+            match pending_verification.binary_search(&para_id) {
+                Ok(_) => return Err(Error::<T>::ParaIdAlreadyPaused.into()),
+                Err(index) => {
+                    pending_verification
+                        .try_insert(index, para_id)
+                        .map_err(|_e| Error::<T>::ParaIdListFull)?;
```

Comment:
It's curious that pausing a container chain will fail when the list of pending container chains is full. I guess it's not a problem because this must be called by root, and root can deregister the pending chains in a batch transaction.

Reply:
Yeah I agree, but what if root tries to pause the container chain without a batch tx? In that case it would fail if we don't handle the situation, and I think it's a good idea to provide a proper error message if that happens.

Reply:
I think the current error is already descriptive enough